torch/utils/_pytree.py

```diff
@@ -151,6 +149,9 @@ def tree_unflatten(values: List[Any], spec: TreeSpec) -> PyTree:
 
     return unflatten_fn(child_pytrees, spec.context)
 
+def tree_map(pytree: PyTree, fn: Callable[[Any], Any]) -> PyTree:
+    flat_args, spec = tree_flatten(pytree)
+    return tree_unflatten([fn(i) for i in flat_args], spec)
```

Comment:
Can you add a test to https://github.com/pytorch/pytorch/blob/master/test/test_pytree.py?